src/miner.cpp

```diff
@@ -473,6 +473,9 @@ void static BitcoinMiner(CWallet *pwallet)
     );
 
     try {
+        unique_ptr<equi> peq;
+        if (solver == "tromp")
+            peq.reset(new equi(1));
```

Comment:
`unique_ptr<equi> peq;` fails to compile with this g++ 4.9.1, and that's despite of `CXXFLAGS` including `-std=c++11`. I didn't look into it for real. What version of gcc/g++ do you test with?